project-2-e-commerce/src/component/form-group/index.js

```diff
@@ -0,0 +1,19 @@
+import './form-group.scss'
+
+function FromGroup({ type, placeholder }) {
```

Comment:
```suggestion
function FormGroup({ type = 'text', placeholder = '' }) {
```
Bổ sung thêm cả default value nhé anh, tránh trường hợp quên mất props nào đó